dex/farm-with-locked-rewards/tests/farm_with_locked_rewards_setup/mod.rs

```diff
@@ -212,6 +212,13 @@ where
             &farming_token_roles[..],
         );
 
+        let reward_token_roles = [EsdtLocalRole::Mint];
+        b_mock.set_esdt_local_roles(
+            energy_factory_wrapper.address_ref(),
+            REWARD_TOKEN_ID,
+            &reward_token_roles[..],
+        );
```

Comment:
Maybe do this only in the respective test. I was quite confused at first why you need such a role in the first place.

Reply:
Updated.